Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -181,8 +183,51 @@ enum PackageShow {
                         source: SiteURL.package(.value(model.repositoryOwner),
                                                 .value(model.repositoryName),
                                                 .readme).relativeURL(),
-                        // Until the content is loaded, substitute a spinner.
-                        .spinner()
+                        .group(
+                            .class("package_tab"),
+                            // Until the content is loaded, substitute a spinner.
+                            .spinner()
+                        )
+            )
+        }
+        
+        func releaseSection() -> Node<HTML.BodyContext> {
+            .turboFrame(id: "releases",
+                        source: SiteURL.package(.value(model.repositoryOwner),
+                                                .value(model.repositoryName),
+                                                .releases).relativeURL(),
+                        .group(
+                            .class("package_tab"),
+                            // Until the content is loaded, substitute a spinner.
+                            .spinner()
+                        )
+            )
+        }
+        
+        func tabBar() -> Node<HTML.BodyContext> {
+            .group(
+                .hr(
+                    .class("minor")
+                ),
+                .section(
```

Comment:
It's definitely semantically valid to put the README and the Release Notes into a `section`, but I'm not sure about the tab bar itself. I think this could just be the `ul` on its own, with a class that indicates a tab bar.